buildscripts/installer/windows/vaultClient.nsi

```diff
@@ -0,0 +1,257 @@
+;--------------------------------
+;Includes
+
+  !include "MUI2.nsh"
+
+;--------------------------------
+;Defines
+
+  !define SHCNE_ASSOCCHANGED 0x8000000
+  !define SHCNF_IDLIST 0
+
+  !define REG_UNINST_KEY "Software\Microsoft\Windows\CurrentVersion\Uninstall\Euclideon udStream"
+  !define /ifndef BUILD_SUFFIX ""
+
+  !getdllversion "..\..\..\builds\udStream${BUILD_SUFFIX}.exe" VersionExpv_
+  !define /ifndef VERSION '${VersionExpv_1}.${VersionExpv_2}.${VersionExpv_3}.${VersionExpv_4}'
+  !define /ifndef VER_MAJOR ${VersionExpv_1}
+  !define /ifndef VER_MINOR ${VersionExpv_2}
+  !define /ifndef VER_REVISION ${VersionExpv_3}
+  !define /ifndef VER_BUILD ${VersionExpv_4}
+
+;--------------------------------
+;General
+
+  ;Name and file
+  Name "udStream"
+  OutFile "udStreamSetup.exe"
+  Unicode True
+
+  ;Default installation folder
+  InstallDir "$PROGRAMFILES64\Euclideon\udStream"
+  
+  ;Get installation folder from registry if available
+  InstallDirRegKey HKCU "Software\Euclideon\udStream" ""
+
+  ;Request application privileges for Windows Vista
+  RequestExecutionLevel admin
+
+;--------------------------------
+;Header Files
+
+  ;!include "WordFunc.nsh"
+
+;--------------------------------
+;Interface Settings
+
+  !define MUI_ABORTWARNING
+
+  !define MUI_ICON "..\..\..\icons\EuclideonClient.ico"
+  !define MUI_UNICON "..\..\..\icons\EuclideonClient.ico"
+
+;--------------------------------
+;Pages
+
+  !insertmacro MUI_PAGE_LICENSE "..\License.txt"
+  !insertmacro MUI_PAGE_COMPONENTS
+  !insertmacro MUI_PAGE_DIRECTORY
+  !insertmacro MUI_PAGE_INSTFILES
+  
+  !insertmacro MUI_UNPAGE_CONFIRM
+  !insertmacro MUI_UNPAGE_INSTFILES
+
+;--------------------------------
+;Languages
+
+  !insertmacro MUI_LANGUAGE "English"
+
+;--------------------------------
+;Version information
+
+  !ifdef VER_MAJOR & VER_MINOR & VER_REVISION & VER_BUILD
+  VIProductVersion ${VER_MAJOR}.${VER_MINOR}.${VER_REVISION}.${VER_BUILD}
+  VIFileVersion ${VER_MAJOR}.${VER_MINOR}.${VER_REVISION}.${VER_BUILD}
+  VIAddVersionKey "ProductName" "udStream Setup"
+  VIAddVersionKey "Company" "Euclideon Pty Ltd"
+  VIAddVersionKey "FileVersion" "${VERSION}"
+  VIAddVersionKey "ProductVersion" "${VERSION}"
+  VIAddVersionKey "FileDescription" "udStream Setup"
+  VIAddVersionKey "LegalCopyright" "� Euclideon Pty Ltd"
+  !endif
+
+;--------------------------------
+;Init function
+Function .onInit
+  System::Call 'kernel32::CreateMutex(i 0, i 0, t "udStreamInstallerMutex") i .r1 ?e'
+  Pop $R0
+  StrCmp $R0 0 +3
+    MessageBox MB_OK|MB_ICONEXCLAMATION "The installer is already running."
+    Abort
+FunctionEnd
+
+;--------------------------------
+;Installer Sections
+
+Section "udStream" SecCore
+  SectionInstType RO
+
+  SetOutPath "$INSTDIR"
+  
+  ;Add files
+  File ..\..\..\builds\udStream${BUILD_SUFFIX}.exe
+  File ..\..\..\builds\vaultConvertCMD.exe
```

Comment:
Probably `udStreamCMD.exe`?

Reply:
`udStreamCMD.exe` or `udStreamConvertCMD.exe`?

Reply:
`Convert` is better

Reply:
Updated.